### PR TITLE
Add Homer

### PR DESCRIPTION
**Pull request by shahvirb**
# Pull request

**Purpose**
Adding Homer as a new application
https://github.com/bastienwirtz/homer

**Approach/Testing**
1. After installing DockSTARTer on my x86_64 system I cloned my fork of the DockSTARTer repo and created the following symlinks. These links allowed me to use "ds -c" in the CLI and the ds GUI to test like a production user
`~/.docker/docs -> <my_repo>/docs`
`~/.docker/compose/.apps -> <my_repo>/compose/.apps`

1. Used https://markdownlivepreview.com/ to preview my homer.md doc.
2. Verified in the GUI that Homer shows up with value of com.dockstarter.appinfo.description
3. Used `ds -a homer` and `ds -r homer` with `ds -c` for remaining testing
4. Verified that HOMER_INIT_ASSETS can be toggled to 1 and 0 and behavior as expected by inspecting container logs.
5. Verified that HOMER_PORT_HTTP can be set and homepage served on this port
6. Verified that HOMER_SUBFOLDER can be set and homepage served on this subfolder path

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] I want to double check that the best practice in DockSTARTer is use of `${DOCKERCONFDIR}/homer:/www/assets` as config/assets volume  instead of `${HOMER_ASSETS}:/www/assets` which was my first instinct and felt more natural to me

**Learning**
Read docs:
1. https://hub.docker.com/r/b4bz/homer 
7. https://github.com/bastienwirtz/homer

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/homer/homer.labels.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+services:
+  homer:
+    labels:
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.description: A dead simple static HOMepage for your servER
+      com.dockstarter.appinfo.nicename: Homer
+      com.dockstarter.appvars.homer_enabled: "false"
+      com.dockstarter.appvars.homer_init_assets: "1"
+      com.dockstarter.appvars.homer_network_mode: ""
+      com.dockstarter.appvars.homer_port_http: "8080"
```
Comment (nemchik) [inline compose/.apps/homer/homer.labels.yml:10]:
```suggestion
      com.dockstarter.appvars.homer_port_8080: "8080"
```
The variable (label) name cadence is `<app>_PORT_<default-port>`.

File: compose/.apps/homer/homer.ports.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+services:
+  homer:
+    ports:
+      - ${HOMER_PORT_HTTP}:8080
```
Comment (nemchik) [inline compose/.apps/homer/homer.ports.yml:4]:
```suggestion
      - ${HOMER_PORT_8080}:8080
```

File: compose/.apps/homer/homer.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    logging:
+      driver: json-file
+      options:
+        max-file: ${DOCKERLOGGING_MAXFILE}
+        max-size: ${DOCKERLOGGING_MAXSIZE}
+    restart: ${HOMER_RESTART}
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERSTORAGEDIR}:/storage
+      - ${DOCKERCONFDIR}/homer:/www/assets
```
Comment (nemchik) [inline compose/.apps/homer/homer.yml:19]: To answer your question, this is good.

**Review by nemchik — CHANGES_REQUESTED** [review]

**Comment by nemchik** [comment]
The link check failed. Can you add the link to the mkdocs.yml in the top level of the repo?

**Comment by shahvirb** [comment]
@nemchik 
> The link check failed. Can you add the link to the mkdocs.yml in the top level of the repo?
Sure thing.

How come the other checks aren't running?

```
3 workflows awaiting approval
First-time contributors need a maintainer to approve running workflows. [Learn more.](https://docs.github.com/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks) 3 successful and 8 expected checks
```

**Comment by nemchik** [comment]
> @nemchik
> 
> > The link check failed. Can you add the link to the mkdocs.yml in the top level of the repo?
> > Sure thing.
> 
> How come the other checks aren't running?
> 
> ```
> 3 workflows awaiting approval
> First-time contributors need a maintainer to approve running workflows. [Learn more.](https://docs.github.com/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks) 3 successful and 8 expected checks
> ```

I had to approve them to run. Security precaution.

**Review by nemchik — APPROVED** [review]

**Pull request merged**